docs/articles/2023/20230908-out_of_core_methods.md

```diff
@@ -0,0 +1,133 @@
+# Memory-efficient implementations of commonly used single-cell methods
+
+*Published:* *September [DAY], 2023*
+
+*By:* *[Pablo Garcia-Nieto](pgarcia-nieto@chanzuckerberg.com)*
+
+The Census team is thrilled to officially announce the first roll out of memory-efficient implementations of some of the most widely used single-cell algorithms.
+
+With just a few lines of code, using the Census Python API, users can now perform the following processing tasks in a regular laptop with 8GB of memory across dozens of millions of cells:
+
+* Calculating average and variance gene expression for cells or genes. See this [tutorial](../../notebooks/experimental/mean_variance.ipynb).
+* Obtaining batch-corrected highly variable genes. See this [tutorial](../../notebooks/experimental/highly_variable_genes).
+
+These implementations are interwoven with the way users query slices of Census data, which means that these tasks can be seamlessly applied to any slice of the 33M+ cells available in Census.
+
+Continue reading for more implementation details and usage examples.
+
+## Efficient calculation of average and variance gene expression across millions of cells
+
+With `cellxgene_census.experimental.pp.mean_variance` users can now get gene expression average and variance for all genes or cells in a given Census query.
+
+### How it works
+
+Calculations are done in an accumulative chunked fashion, meaning that only a small fraction of the total data is processed at any given time.
```

Comment:
Very tiny point of clarity - is "chunked" commonly understood as the same as "incremental" or "step-by-step"? I think it may be slightly easier to understand if you use "...an accumulative, step-by-step fashion, ..."